account_invoice_pricelist/README.rst

```diff
@@ -0,0 +1,86 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================
+Pricelist on Invoices
+=====================
+
+* Add a stored field pricelist on invoices, related to the partner pricelist;
+* Possibility to group by pricelist on account.invoice view;
+
+.. image:: static/src/description/screenshot_group_by.png
+
+
+This module doesn't add real feature by it self for end-users, but is usefull
+to do reporting, in a inherited module.
+
+Installation
+============
+
+Classical installation.
+
+Configuration
+=============
+
+Nothing to do.
+
+Note
+====
+
+* The field will not be stored for purchase invoices as long as 'purchase'
```

Comment:
Ah, OK, I didn't understand what you mean. Of course, you should rephrase it and put it "Installation" section something like that:
```
Installation
============

Current invoices will be filled with the corresponding partner pricelist when this module is installed, but supplier invoices and refund don't if you don't have already installed purchase module previously, as there is no supplier pricelist on the partner if you don't install it.
```